### ISSUE TITLE
Unclosed ES6 template error with RegExp

### ISSUE THREAD

**Issue by jkirkwood** [riot/compiler#114]
See this jsfiddle: http://jsfiddle.net/5r6oj3pm/1/

The `$` at the end of the regex is causing an "Unclosed ES6 template" error during compilation:

```js
this.value = this.value.replace(new RegExp(`[^0-9.-]+$`, 'g'), '');
```

If I remove the `$` everything works fine.

**Comment by GianlucaGuarini** [comment]
For now just use the literal constructor https://next.plnkr.co/edit/FIqN5yUK7ZEkWfkn and everything should be fine. The upcoming riot@4 version has already solved these kind of issues with a real tag parser

**Comment by GianlucaGuarini** [comment]
Fixed with the new compiler v4.0.0. Unfortunately the old compiler will not be able to handle this issue
